commcare-cloud-bootstrap/environment/inventory.ini.j2

```diff
@@ -11,4 +11,10 @@
 {%- endfor %}
 {% endfor %}
 
+{% for _, group in inventory.all_groups.items()|sort %}
+[{{ group.name }}:vars]
+hostname='{{ group.name }}'
+{{ host_name }}
+{%- endfor %}
+
```

Comment:
Oh, you know what? I think there was a better way to do this that wouldn't cause https://github.com/dimagi/commcare-cloud/pull/1883. There are already places in `commcare_cloud_bootstrap.py` where we add variables, and then those are added to this template in the line above
```
{%- for key, value in machine.vars.items()|sort %} {{ key }}={{ value }}{% endfor %}
```

Reply:
I jus tested it again without my change, and the lines that looks like 
```
[celery:vars]
hostname='celery'
```
don't show up. I can look into why that is!

Reply:
Oh yep, that works.. I'm not sure why it didn't work the first time I had this, maybe I just didn't pull from the latest master or something.

Reply:
https://github.com/dimagi/commcare-cloud/pull/1884